From 8dcb05d0296b3f5e5114e188a242c26d09af4b84 Mon Sep 17 00:00:00 2001
From: Christopher Albert <albert@tugraz.at>
Date: Wed, 22 Apr 2026 12:28:03 +0200
Subject: [PATCH 11/23] testsuite: Add Bugzilla reporter attribution to
 gomp/pr101760 and pr102314 [PR fortran/101760, PR fortran/102314]

Retrospective attribution header added as part of a manual
Bugzilla-provenance audit on 2026-04-22 (meta-repo tracking commit
40f0a94; GitHub issues on krystophny/gcc-dev).  The existing GCC
Fortran testsuite convention (see pr113363.f90, pdt_50.f03,
arrayio_4.f90) uses an in-file "! Contributed by ..." line; practice
is inconsistent and attribution is not strictly required for small
Bugzilla reducers, but we are adding it retroactively for provenance
safety and completeness.  No behavioural change.

Assisted-by: Claude (Anthropic)
Assisted-by: GPT (OpenAI)

	PR fortran/101760
	PR fortran/102314

gcc/testsuite/ChangeLog:

	* gfortran.dg/gomp/pr101760.f90: Add reporter attribution header.
	* gfortran.dg/gomp/pr102314.f90: Add reporter attribution header.

Signed-off-by: Christopher Albert <albert@tugraz.at>
---
 gcc/testsuite/gfortran.dg/gomp/pr101760.f90 | 1 +
 gcc/testsuite/gfortran.dg/gomp/pr102314.f90 | 1 +
 2 files changed, 2 insertions(+)

diff --git a/gcc/testsuite/gfortran.dg/gomp/pr101760.f90 b/gcc/testsuite/gfortran.dg/gomp/pr101760.f90
index ab7bf515a8c..b81be1ec90f 100644
--- a/gcc/testsuite/gfortran.dg/gomp/pr101760.f90
+++ b/gcc/testsuite/gfortran.dg/gomp/pr101760.f90
@@ -1,4 +1,5 @@
 ! PR fortran/101760
+! Contributed by G. Steinmetz  <gscfq@t-online.de>
 ! { dg-do compile }
 ! { dg-options "-fopenmp -O1" }
 
diff --git a/gcc/testsuite/gfortran.dg/gomp/pr102314.f90 b/gcc/testsuite/gfortran.dg/gomp/pr102314.f90
index 6e8a45487f3..923fcf7527c 100644
--- a/gcc/testsuite/gfortran.dg/gomp/pr102314.f90
+++ b/gcc/testsuite/gfortran.dg/gomp/pr102314.f90
@@ -1,4 +1,5 @@
 ! PR fortran/102314
+! Contributed by G. Steinmetz  <gscfq@t-online.de>
 ! { dg-do compile }
 ! { dg-options "-fopenmp -O2" }
 
-- 
2.53.0

